### PR TITLE
Upgraded issue templates to use GitHub issue forms

### PR DESCRIPTION
**Pull request by vendz**
a pull request for #18

### CODE REVIEW

**Comment by lohanidamodar** [comment]
@vendz  Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

**Comment by vendz** [comment]
@lohanidamodar requested changes have been made. 
kindly review it!

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: dropdown
+    id: svelte-sdk-version
+    attributes:
+      label: "🎲 Svelte SDK version"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml]: Let's also keep Appwrite version here, as it's relevant.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Great work @vendz. Small request to make it even better 🚀

**Comment by vendz** [comment]
@lohanidamodar changes have been made.
kindly review it!

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: dropdown
+    id: appwrite-version
+    attributes:
+      label: "🎲 appwrite version"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml]:
```suggestion
      label: "🎲 Appwrite version"
```

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      description: "What version of Appwrite are you running?"
+      options:
+        - Version 0.2.x (beta)
+        - Version 0.1.x (beta)
+        - Version 0.0.1 (beta)
+        - Version 0.0.1 (alpha)
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml]: The versions should be Appwrite versions.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Few more changes @vendz !

**Comment by vendz** [comment]
@lohanidamodar please review and merge it.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU so much! All changes merged 🥳
Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swag!

**Pull request merged**